security/backends/elasticsearch/writer.py

```diff
@@ -214,42 +225,38 @@ def _get_logger_message(self, logger, last_version=False, **extra_data):
         logger_data.update(extra_data)
 
         logger.backend_logs['last_elasticsearch_version'] = version
-        return self._get_log_message(
+        return self._get_log_data(
             logger.id,
             logger.logger_name,
             version,
             logger_data
         )
 
     def create_or_update_index_from_logger(self, logger, is_last=False, **extra_data):
-        logstash_logger.info(
-            self._get_logger_message(
-                logger,
-                last_version=is_last,
-                **extra_data
-            )
-        )
+        logger_data = self._get_logger_data(logger, last_version=is_last, **extra_data)
+        if settings.ELASTICSEARCH_LOGSTASH_WRITER:
+            logstash_logger.info(self._format_log_message(**logger_data))
+        if settings.ELASTICSEARCH_VECTOR_WRITER:
+            vector_logger.info("", **logger_data)
```

Comment:
why is empty string here?

Reply:
Logger expects as a first argument event/message. I guess that in our case, the message should be either empty or we should use `index_name` as a message. 

I chose to have `index_name` as a separate field so I don't have anything to include in the message.